spark/src/test/scala/geotrellis/spark/io/LayerUpdateSpaceTimeTileSpec.scala

```diff
@@ -65,12 +65,21 @@ trait LayerUpdateSpaceTimeTileSpec
         updater.update(layerId, sample)
       }
 
+      it("should overwrite a layer") {
+        updater.overwrite(layerId, sample)
```

Comment:
Does this actually confirm that the update happened?

Reply:
It only confirms it didn't through, its possible that that an update was empty, either in bounds or in records in which case it would be NOOP. 

Overall I agree it would be much more helpful if this returned number of records updated/written but that would require changing interface of `_RDDWriter.write`... or (I'm realizing as I'm typing this) to add a new counting method and wrap it to preserve the interface.

I would push this change off to `2.0` though, what do you think?